### ISSUE TITLE
Snippet Streamfield help text covers controls

### ISSUE THREAD

**Issue by heymonkeyriot** [wagtail/wagtail#2792]
In the snippet model:

```
body = StreamField([
        ('heading', blocks.CharBlock(classname="full title")),
        ('paragraph', blocks.RichTextBlock()),
        ('image', ImageChooserBlock()),
        ('table', TableBlock(icon="fa-table"))
    ], help_text='Enter the body content for the FAQ')
```

On the snippet page
`<div class="object-help help">Enter the body content for the FAQ</div>` blocks the delete icon for the first item.

Since the help text isn't displayed the trivial solution is to delete the help_text from the model, but that isn't ideal since the client needs the help text. Have navigated around this with custom CSS for this project, but it'd be good for help text to not break the controls :)

Note, the class being used on the snippet for help text is different to the class used on a normal page. On normal pages adding help text to the streamfield doesn't cover the controls (though of course the help text is still not visible).

**Comment by strindhaug** [comment]
A fairly straight forward fix is to add this css to the admin interface:

```css
/* fix bug with streamfield help text: */
.object.stream-field .object-help {
  position: relative; /* counteracts the default absolute position */
}
.object.stream-field:hover .object-help {
  opacity: 1; /* makes the helptext show up when hovering the streamfield */
}
```

The only slight disadvantage with this fix is that it adds a bit extra space between the pink title bar and the stream field content.

Until this is fixed in wagtail you can fix it by [overriding the wagtail admin base template](http://docs.wagtail.io/en/v1.7/advanced_topics/customisation/branding.html) by creating a template called `<your-project-template-dir>/wagtailadmin/base.html` like this:

```django
{% extends "wagtailadmin/base.html" %}
{% block css %}
  {{ block.super }}
  <style media="screen">
  /* fix bug with streamfield help text: */
  .object.stream-field .object-help {
    position: relative; /* counteracts the default absolute position */
  }
  .object.stream-field:hover .object-help {
    opacity: 1; /* makes the helptext show up when hovering the streamfield */
  }
  </style>
{% endblock %}
```

**Comment by gasman** [comment]
Fixed in #3168
